inspec.yml

```diff
@@ -40,6 +40,7 @@ inputs:
       - "SV-230365" # IA-5 (1) d
       - "SV-230366" # IA-5 (1) d
       - "SV-230367" # IA-5 (1) d
+      - "SV-230368" # IA-5 (1) d
```

Comment:
I'm not sure if it's `d`. In ARS, it seems to be `h`. @ejaronne?

Reply:
done